tap_github/repository_streams.py

```diff
@@ -891,7 +891,16 @@ def get_records(self, context: Optional[Dict] = None) -> Iterable[Dict[str, Any]
             self.logger.debug(f"No comments detected. Skipping '{self.name}' sync.")
             return []
 
-        return super().get_records(context)
+        # Getting issue comments is sometimes too expensive on large repos and results server errors.
```

Comment:
```suggestion
        # Getting issue comments is sometimes too expensive on large repos and results in server errors.
```

Reply:
Maybe worth linking to the relevant github api docs?

Reply:
I could not really find a github doc for that, did you?